@nav-frontend/icons/svg/Data-1.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="Data">
```

Comment:
Gjelder forsåvidt alle ikonene tipper jeg, men denne virket fin å kommentere på. :) 

Skulle man prøvd å fjerne `id="Data"` osv fra filene? Det har sikkert ikke så mye å si, men blir litt mindre filer, og vi minsker muligheten for at de blir stylet uten at det er meningen. 

Videre, hva tenker dere om å bruke `currentColor` istedet for hardkodet verdi for `fill` og `stroke` i svg'ene her.

Reply:
Her er forsåvidt react komponenten som blir generert fra den SVG en. id er fjernet og fill er currentColor
```jsx
function Data1(
  props: React.SVGProps<SVGSVGElement>,
  svgRef?: React.Ref<SVGSVGElement>
) {
  return (
    <svg
      width="1em"
      height="1em"
      viewBox="0 0 24 24"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      ref={svgRef}
      {...props}
    >
      <g fill="currentColor">
        <path d="M0 22h24v2H0z" />
        <path
          d="M2 13h4v6H2v-6zM10 1h4v18h-4V1zM18 7h4v12h-4V7z"
          stroke="currentColor"
          strokeWidth={2}
        />
      </g>
    </svg>
  );
}

const ForwardRef = React.forwardRef(Data1);
```

Reply:
Den ser jo super ut den der. :+1: 

Men inkluderer vi bare react-komponentene når vi publiserer, eller er original svg'ene også med? Hvis vi bare eksponerer react-komponentene så tenker det jeg er helt innafor sånn som nå. `--replace-attr-values '#3E3832=currentColor'` er kanskje ikke den mest robuste løsningen, men det fungerer for nå. 

Men om vi også inkluderer selve svg'ene så tenker jeg at de bør være like.

Reply:
Denne løsningen vil eksponere react som default, men alle svg filene følger med så `@nav-frontend/icons/svg/data1.svg` vil fungere. Enig i at de bør være like, kan se etter løsninger for å bearbeide selve svg filene hentet fra Figma.